storage/database/db_manager.go

```diff
@@ -379,13 +383,11 @@ func partitionedDatabaseDBManager(dbc *DBConfig) (*databaseManager, error) {
 			fallthrough
 		case StateTrieDB:
 			newDBC := getDBEntryConfig(dbc, entryType, dir)
-			if dbc.NumStateTriePartitions > 1 {
-				db, err = newPartitionedDB(newDBC, entryType, dbc.NumStateTriePartitions)
-			} else {
-				db, err = newDatabase(newDBC, entryType)
-			}
+			newDBC.DBType = DynamoDB
```

Comment:
```suggestion
			// TODO-Klaytn: parameterize DB type (leveldb, dynamodb,memorydb, ...)
```
This is very risky to be in `dev` branch`

Reply:
I reverted the change